dashboard/config/linux/upstream-arm-full.config

```diff
@@ -2098,6 +2108,7 @@ CONFIG_PREVENT_FIRMWARE_BUILD=y
 #
 CONFIG_FW_LOADER=y
 # CONFIG_FW_LOADER_DEBUG is not set
+# CONFIG_RUST_FW_LOADER_ABSTRACTIONS is not set
```

Comment:
@a-nogikh there are a few RUST related configs:
1. CONFIG_RUST_FW_LOADER_ABSTRACTIONS
2. CONFIG_BLK_DEV_RUST_NULL
3. CONFIG_RUST_PHYLIB_ABSTRACTIONS

They probably deserve some analysis and be enabled for the rust config.